lib/context/pin-context.tsx

```
@@ -0,0 +1,381 @@
+        }),
+      };
+    case "ADD_PENDING_OPERATION":
+      return {
+        ...state,
+        pendingOperations: new Set([
+          ...state.pendingOperations,
+          action.payload,
+        ]),
+      };
+    case "REMOVE_PENDING_OPERATION":
+      const newPendingOperations = new Set(state.pendingOperations);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)